README.md

```diff
@@ -54,8 +54,7 @@ Photos by [Clay Banks](https://unsplash.com/photos/three-bicycles-parked-in-fron
 and [Derrick Cooper](https://unsplash.com/photos/brown-road-in-forest-during-daytime-L505cPnmIds).
 
 Try a live demo of this theme by running
-`nix run github:danth/stylix#testbed:gnome:default:light` or
-`nix run github:danth/stylix#testbed:gnome:default:dark`.
+`nix run github:danth/stylix#testbed:gnome:default:light:image:scheme`.
```

Comment:
It might be safe to assume that most applications use a light theme by default. To make the Stylix functionality more obvious, we might want to promote the dark theme:

```suggestion
`nix run github:danth/stylix#testbed:gnome:default:dark:image:scheme`.
```

Reply:
I actually had this thought myself right after opening the PR :) Will change it